src/main/java/bio/terra/cli/apps/DockerAppsRunner.java

```diff
@@ -222,10 +259,7 @@ private String startDockerContainerWithTerraInit(
    * @throws RuntimeException if the local directory does not exist or is not a directory
    */
   private String startDockerContainer(
```

Comment:
Should this only ever be called from `startDockerContainerWithTerraInit`?

Reply:
Yes. I was just separating out the Docker-specific part of starting a container from the Terra CLI-specific part. Per our conversation above about moving some of these methods into a helper class, I think the startDockerContainer would go into a DockerHelper class, and the ..WithTerraInit would stay here.